charts/rstudio-workbench/README.md

```diff
@@ -413,6 +413,54 @@ launcher:
           effect: "NoSchedule"
 ```
 
+## Chronicle Agent
+
+This chart supports use of a sidecar Chronicle agent to report data to a Chronicle server. The agent can be enabled
+by setting `chronicleAgent.enabled=true`.
+
+By default, the chart will attempt to lookup an existing Chronicle server deployed in the release namespace. The
+searched namespace can be changed setting `chronicleAgent.serverNamespace`. If a server exists, it will set the
+Chronicle agent's server value to the server's service name and will use an agent version to match the server version.
+This auto-discovery behavior can be disabled by setting `chronicleAgent.autoDiscovery=false`.
+
+To set the server address and/or version manually, set the following values:
+```yaml
+chronicleAgent:
+  enabled: true
+  serverAddress: <server-address>
+  image:
+    tag: <agent-version>
+```
+
+If preferred, the Chronicle agent can be directly defined as a sidecar container using either `initContainers`
+(recommended) or `sidecar` values. Below is an example of directly defining the Chronicle agent as a native sidecar
+container using `initContainers`:
+```yaml
+initContainers:
+  - name: chronicle-agent
+    restartPolicy: Always
+    image: ghcr.io/rstudio/chronicle-agent:<agent-version>
+    env:
+      - name: CHRONICLE_SERVER_ADDRESS
+        value: "http://<address>"
+```
+
+For more information on Posit Chronicle, see the [Chronicle documentation](https://docs.posit.co/chronicle/).
+
+### Chronicle Workbench API Key
+
+In most cases where the Chronicle agent is deployed as a sidecar to Workbench, the agent can provision its own API key.
```

Comment:
I don't think this is true, but maybe I've missed something. Do we automatically generate an API key as part of this chart now?

Reply:
Perhaps I misunderstand the process here? I thought Chronicle magically provisioned its own API key. Does it simply not require one for Workbench in most cases for this setup with a sidecar?

Reply:
Yeah, sorry we should probably have been more explicit about this. Since [the Workbench API is still a "preview" feature](https://docs.posit.co/ide/server-pro/admin/workbench_api/workbench_api.html), we don't require it to be enabled in order to run Chronicle. Instead, if a user doesn't supply a Workbench API key, we just don't turn on Workbench API scraping in the agent.